request.go

```diff
@@ -140,6 +140,8 @@ func allocateBody(key, version int16) protocolBody {
 		return &DescribeConfigsRequest{}
 	case 33:
 		return &AlterConfigsRequest{}
+	case 36:
```

Comment:
@varun06 should we use the `APIKeySASLAuth` const defined in `SaslAuthenticateRequest` here? Seems like the Right Thing To Do, but also breaks with the style of the rest of the case statements.

Reply:
It does, but for now we can keep it this way for the sake of consistency, and come back later.